MSFragger-GUI/src/com/dmtavt/fragpipe/cmd/CmdMSBooster.java

```diff
@@ -162,15 +155,15 @@ public boolean configure(Component comp,
         bufferedWriter.write("renamePin = 1\n");
         bufferedWriter.write("useRT = " + (predictRT ? "true" : "false") + "\n");
         bufferedWriter.write("useSpectra = " + (predictSpectra ? "true" : "false") + "\n");
-        bufferedWriter.write("useIM = " + ((hasTimsTof && predictIm) ? "true" : "false") + "\n");
+        bufferedWriter.write("useIM = " + (predictIm ? "true" : "false") + "\n");
         bufferedWriter.write("fragger = " + fraggerParams + "\n");
         bufferedWriter.write("deletePreds = false\n"); // FragPipe-PDV need the prediction files.
         bufferedWriter.write("rtModel = " + rtModel + "\n");
         bufferedWriter.write("spectraModel = " + spectraModel + "\n");
         bufferedWriter.write("imModel = " + imModel + "\n");
         bufferedWriter.write("findBestRtModel = " + (findBestRtModel ? "true" : "false") + "\n");
         bufferedWriter.write("findBestSpectraModel = " + (findBestSpectraModel ? "true" : "false") + "\n");
-        bufferedWriter.write("findBestImModel = " + ((hasTimsTof && findBestImModel) ? "true" : "false") + "\n");
+        bufferedWriter.write("findBestImModel = " + (findBestImModel ? "true" : "false") + "\n");
```

Comment:
There is no need to set useIM = true for non-timsTOF data, right?

Reply:
Refer to discussion on uncalibrated mzml

Reply:
The uncalibrated mzml is not supposed to be used as the FragPipe input. It is more like an intemediate file.

We also not recommend converting .d to mzML.

Reply:
added back in latest commit